kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -3488,6 +3492,22 @@ bool IndexerASTVisitor::VisitFunctionDecl(clang::FunctionDecl* Decl) {
         RangeInCurrentContext(Decl->isImplicit(), DeclNode, DefinitionRange);
     MaybeRecordFullDefinitionRange(DefinitionRangeInContext, DeclNode,
                                    BuildNodeIdForDefnOfDecl(Decl));
+
+    // Record a childof edge from the anchor spanning the lambda's definition to
+    // the semantic node representing the enclosing function.
+    if (absl::GetFlag(FLAGS_emit_anchor_scopes) && lambda &&
+        DefinitionRangeInContext.has_value()) {
+      if (Job->BlameStack.size() < 2) {
+        // TODO(justbuchanan): add childof edge from the lambda to the file node
+      } else {
+        // The last entry in the blame stack is the lambda we're currently
+        // looking at. We need to go one up from that to find the enclosing
+        // function
+        for (const auto& Caller : Job->BlameStack.rbegin()[1]) {
+          Observer.recordChildOfEdge(DefinitionRangeInContext.value(), Caller);
```

Comment:
Just `*DefinitionRangeInContext` after checking for validity (as is done above).

Reply:
Done